datastore/mongodb/filter_outputs.go

```diff
@@ -0,0 +1,28 @@
+package mongodb
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-cantabular-filter-flex-api/model"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+// CreateFilterOutput creates a new FilterOutputs in the CantabularFilters colllection
+func (c *Client) CreateFilterOutput(ctx context.Context, f *model.FilterOutputResponse) error {
```

Comment:
Ah I see you've create a FilterOutputResponse struct in package model and used it here as a database model.. The FilterOutputResponse type should be in package api contract.go, and should be only used for responding to the request. The database model should be in package model as you have it but it shouldn't be called FilterOutputResponse, it should be called FilterOutput. If the fields are identical you can embed FilterOutput in FilterOutputResponse as is done with createFilterRequest.
Also it should be `getFilterOutputResponse`.